server/routes/specialFeaturesDD.router.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
```

Comment:
What's the "DD" at the end of this file name stand for?

Reply:
Ah, is it DropDown?

Rule of thumb -- assume your server doesn't know how the client is using it. That makes it more flexible, if you need to change client behavior later, and keeps your code better organized and maintainable.

In other words, this endpoint is just returning a list of special features, it doesn't care if you want to use them in a dropdown, render them in a table, or what... Just call it `specialFeatures`